tomviz/AnimationHelperDialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>699</width>
-    <height>448</height>
+    <height>449</height>
```

Comment:
Any reason for the height increase or was is just designer doing its thing ?

Reply:
I think it was just the designer doing its thing.